docs/how-to/how-to-manage-follow-up-issues.md

```diff
@@ -0,0 +1,18 @@
+# Guidelines for Managing Follow-Up Issues
+
+This document provides clear guidelines on how to effectively handle follow-up issues identified during events such as Technical Sprint Reviews.
+
+## Rules
+1. **Create Follow-Up Issues:**
```

Comment:
I wouldn't use bold font in this way. Please have a look at the [guidelines for using bold font](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/03-formatting.md#items-written-in-bold-font) and adjust the whole doc.

Reply:
fixed